bot/src/index.js

```diff
@@ -22,7 +22,6 @@ const app = new App({
     signingSecret: process.env.SIGNING_SECRET,
     socketMode: true,
     appToken: process.env.APP_TOKEN,
-    logLevel: LogLevel.TRACE,
```

Comment:
Is this the default? Typicallky i've seen a default of INFO or ERROR and then people setting an environment variable for DEBUG or TRACE

Reply:
Do you need a default? I had trace which is an invalid option based on the [docs](https://slack.dev/bolt-js/concepts/logging) so I thought to just remove it. It seems to still work, but may not be beat practice.